### PR TITLE
Add patreon button

### PR DESCRIPTION
**Pull request by dbrgn**
It ain't pretty, but we can still improve the design later on.

patreon

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 themes
+*.swp
```
Comment (ubruhin) [inline .gitignore]: Could you maybe move this into a separate commit? :)

File: static/css/HPstyles.css [hunk 2]
```diff
@@ -292,6 +292,19 @@ footer a:hover {
     border-color: black;
 }
 
+.btn-patreon {
+    color: #e6461a;
+    background-color: #232d32;
```
Comment (ubruhin) [inline static/css/HPstyles.css]: Mmh I don't like the color ;) Maybe the red/orange from Patreons logo would look better?

Comment (dbrgn) [inline static/css/HPstyles.css]: The "color" value is actually the red/orange from the Patreon logo. And 232d32 is their footer color.

Comment (ubruhin) [inline static/css/HPstyles.css]: Oh okay, how does it look if we swap these colors?

**Comment by ubruhin** [comment]
I have [updated](https://github.com/LibrePCB/LibrePCB.github.io/commit/06fca226e477ea26c946607e322567172d6267c6) the `.gitignore` on the `source` branch and rebased this branch, so the `.gitignore` modification is no longer in this PR ;)

And I added a commit to this PR which changes the colors of the button to following:

auswahl_008

Looks better, doesn't it?

**Comment by dbrgn** [comment]
Yep :) Sorry for forgetting about this :)
